src/scales/scale.time.js

```diff
@@ -194,8 +194,8 @@ function toTimestamp(input, options) {
 			: adapter.parse(value);
 	}
 
-	if (value !== null) {
-		return +value;
+	if (helpers.isFinite(value)) {
```

Comment:
The isFinite instead of !== null is good, but is going to be slower (but not in chrome unexpectedly!):
https://jsperf.com/isfinite-vs-null-plus/1

I would leave this also as is, we accept null as a "parse failure" and anything else is regarded (and converted to) a number.

Reply:
Done